Mathlib/LinearAlgebra/Alternating/Basic.lean

```diff
@@ -542,6 +542,19 @@ theorem compLinearMap_inj (f : M₂ →ₗ[R] M) (hf : Function.Surjective f)
     (g₁ g₂ : M [⋀^ι]→ₗ[R] N) : g₁.compLinearMap f = g₂.compLinearMap f ↔ g₁ = g₂ :=
   (compLinearMap_injective _ hf).eq_iff
 
+/-- If two `R`-alternating maps from `R` are equal on 1, then they are equal.
+
+This is the alternating version of `LinearMap.ext_ring`. -/
+@[ext]
+theorem ext_ring {R} [CommSemiring R] [Module R N] [Finite ι] ⦃f g : R [⋀^ι]→ₗ[R] N⦄
+    (h : f (fun _ ↦ 1) = g (fun _ ↦ 1)) : f = g :=
+  coe_multilinearMap_injective <| MultilinearMap.ext_ring h
+
+/-- The only `R`-alternating map from two or more copies of `R` is the zero map. -/
```

Comment:
Optional: add a TODO to add a version assuming `finrank R M < Nat.card ι`. Otherwise LGTM. Thanks!
bors d+

Reply:
I guess you mean this result?
```lean
import Mathlib

universe uι uK uM uN
variable {ι : Type uι} {K : Type uK} {M : Type uM} {N : Type uN}
variable [Ring K] [AddCommGroup M] [Module K M] [AddCommGroup N] [Module K N] [NoZeroSMulDivisors K N]

open Cardinal in
def uniqueOfLtCard [Nontrivial K] (h : lift.{uι} (Module.rank K M) < lift #ι) : Unique (M [⋀^ι]→ₗ[K] N) where
  uniq f := AlternatingMap.ext fun v =>
    f.map_linearDependent v fun hv =>
      lt_irrefl _ <| hv.cardinal_lift_le_rank.trans_lt h
```
this doesn't fit in this file, so I'm not sure if thi is a good place for a TODO either.

Can you find a home for this yourself?

bors merge